intel_owl/settings/celery.py

```diff
@@ -8,9 +8,12 @@
 from .aws import AWS_SQS, AWS_USER_NUMBER
 
 RESULT_BACKEND = "django-db"
-BROKER_URL = get_secret(
-    "BROKER_URL", "sqs://" if AWS_SQS else "amqp://guest:guest@rabbitmq:5672"
-)
+BROKER_URL = get_secret("BROKER_URL", None)
+if not BROKER_URL and AWS_SQS:
+    BROKER_URL = "sqs://"
+else:
+    BROKER_URL = "redis://redis:6379/1"  # 0 is used by channels
+
```

Comment:
```suggestion
if not BROKER_URL:
    if AWS_SQS:
        BROKER_URL = "sqs://"
    else:
        BROKER_URL = "redis://redis:6379/1"  # 0 is used by channels

```
I think you wanted to do that instead

Reply:
I think @0ssigeno  did this